<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QWidget" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../doc/img/logo/NorseVPN.svg</normaloff>../doc/img/logo/NorseVPN.svg</iconset>
  </property>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="config">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <widget class="QLabel" name="label_cyberSec">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>When enabled, the CyberSec feature will automatically block suspicious websites so that no malware or other cyber threats can infect your device. Additionally, no flashy ads will come into your sight.</string>
     </property>
     <property name="text">
      <string>Cyber-Sec</string>
     </property>
    </widget>
    <widget class="QSlider" name="slid_cyberSec">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slid_autoConnect">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_firewall">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>blub</string>
     </property>
     <property name="text">
      <string>Firewall</string>
     </property>
    </widget>
    <widget class="QSlider" name="slid_ipv6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slid_notify">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slid_firewall">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_killSwitch">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This security feature blocks your device from accessing the Internet outside the secure VPN tunnel, in case connection with a VPN server is lost.</string>
     </property>
     <property name="text">
      <string>Kill Switch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_obfuscate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Obfuscate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_notify">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Notify</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_autoconnect">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>When enabled, this feature will automatically try to connect to VPN on operating system startup. </string>
     </property>
     <property name="text">
      <string>Auto Connect</string>
     </property>
    </widget>
    <widget class="QSlider" name="slid_obfuscate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ipv6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>IPv6</string>
     </property>
    </widget>
    <widget class="QSlider" name="slid_killSwitch">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_protocol">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>8</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtBottom</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_technology">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>48</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_technology">
     <property name="geometry">
      <rect>
       <x>188</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Technology</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_protocol">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>13</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="dns">
    <attribute name="title">
     <string>DNS</string>
    </attribute>
    <widget class="QLineEdit" name="line_dns">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>8.8.8.8</string>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
    <widget class="QPushButton" name="dns_delete">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_dns">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>11</y>
       <width>351</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="whitelist">
    <attribute name="title">
     <string>Whitelist</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
